graphitron-codegen-parent/graphitron-java-codegen/src/test/java/no/sikt/graphitron/queries/fetch/TableMethodTest.java

```diff
@@ -62,18 +62,75 @@ void paginated() {
+                                return _iv_ctx
+                                        .select(customerForQuery_customer(first_name))
+                                        .from(_a_customer)
+                                        .where(first_name != null ? _a_customer.FIRST_NAME.eq(first_name) : DSL.noCondition())
+                                        .fetchOne(_iv_it -> _iv_it.into(Customer.class));
+                            }
+                            private static SelectField<Customer> customerForQuery_customer(String first_name) {
+                                var customerTableMethod = new CustomerTableMethod();
+                                var staffTableMethod = new StaffTableMethod();
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                _a_customer = customerTableMethod.customerTable(_a_customer, first_name);
+                                var _a_customer_2168032777_address = _a_customer.address().as("address_2138977089");
+                                var _a_address_2138977089_staff = _a_customer_2168032777_address.staff().as("staff_4083961958");
+                                _a_address_2138977089_staff = staffTableMethod.staffTable(_a_address_2138977089_staff);
+                                return DSL.row(
+                                        _a_customer.getId(),
+                                        DSL.field(
+                                                DSL.select(customerForQuery_customer_d1_staff(first_name))
+                                                .from(_a_customer_2168032777_address)
+                                                .join(_a_address_2138977089_staff)
+                                        )
+                                ).mapping(Functions.nullOnAllNull(Customer::new));
+                            }
+                            private static SelectField<Staff> customerForQuery_customer_d1_staff(String first_name) {
+                                var customerTableMethod = new CustomerTableMethod();
+                                var staffTableMethod = new StaffTableMethod();
+                                var _a_customer = CUSTOMER.as("customer_2168032777");
+                                _a_customer = customerTableMethod.customerTable(_a_customer, first_name);
+                                var _a_customer_2168032777_address = _a_customer.address().as("address_2138977089");
+                                var _a_address_2138977089_staff = _a_customer_2168032777_address.staff().as("staff_4083961958");
+                                _a_address_2138977089_staff = staffTableMethod.staffTable(_a_address_2138977089_staff);
+                                return DSL.row(_a_address_2138977089_staff.getId()).mapping(Functions.nullOnAllNull(Staff::new));
+                            }
+                        """
```

Comment:
```suggestion
                "customer = CUSTOMER.as(",
                "customer = customerTableMethod.customerTable(_a_customer, first_name)",
                "address_2138977089_staff = _a_customer_2168032777_address.staff().as(",
                "address_2138977089_staff = staffTableMethod.staffTable(_a_address_2138977089_staff)"
```
Endringene i denne PR-en skal ikke påvirke dette egentlig, så det som var her før er godt nok.